src/server/index.js

```diff
@@ -249,11 +295,7 @@ nextApp.prepare().then(() => {
         continue;
       }
     }
-    const savedId = await saveProfile(
-      data.profile.login,
-      repositories,
-      excludedRepos,
```

Comment:
Why remove excludedRepos?

Reply:
I removed it because we don't need it anymore, once the  selected repositories are filtered only their files and keys will be saved in `dependecies.json`  making it easy to retrieve at once without further filtering.

Reply:
But that is not part of the permissions right?

Reply:
Yes.